server/app/services/item_tag.py

```diff
@@ -0,0 +1,19 @@
+from typing import List
+
+from ..models.tag import Tag
+from ..models.user import User
+from ..models.item_tag import ItemTag
+from ..schemas.tag import TagCreate
+from ..database import get_session
+
+
+def create_item_items_tags(item_id: int, tags: List[TagCreate], user: User):
```

Comment:
consider changing the function name to something more "Self-explanatory".

Reply:
I think it is "Self-explanatory" but I'm open for suggestions

Reply:
I think 'add_tags_to_item' might be a better option as it describes the functionality more precisely.

Reply:
But it's not what it does

Reply:
Added a fix